apps/bas/Makefile

```diff
@@ -2,6 +2,9 @@
 # a phony dependency that can be used as a dependency to force builds
 FORCE:
 
+install:
+	# sudo apt install -y DEPS
```

Comment:
Why is this commented out? Also same question for the other apps with this haha.

Reply:
These don’t actually need to install anything, but the build script runs the `install` target in all the apps.

Reply:
should we say that in the comment then, for posterity?